content/docs/rules/_index.md

```diff
@@ -76,7 +76,7 @@ As noted above, macros provide a way to define common sub-portions of rules in a
 
 With this macro defined, we can then rewrite the above rule's condition as `in_container and proc.name = bash`.
 
-For many more examples of rules and macros, please take a look `rules/falco_rules.yaml`.
```

Comment:
You can drop the `.html` in link URLs. Not necessary, just FYI.

Reply:
Thanks for the catch. I did that with others, but missed that one.